requirements.txt

```
@@ -1,6 +1,6 @@
 # the default package dependencies
 
-numpy>=1.16.6
+numpy>=1.17
```

`numpy.random.SeedSequence` is only available in 1.17+.
@Borda @carmocca is it acceptable to bump the version?

---

hmm, it doesn't want to compile on our "minimal" CI tasks

---

Seems like it's on python 3.8 and minimal CI for all platforms.

---

let's binary search which version works, starting with:

---

1.17.5 worked -> going lower

```suggestion
numpy>=1.17.3
```